pinot-common/src/main/java/org/apache/pinot/common/utils/PinotDataType.java

```diff
@@ -412,6 +577,16 @@ public double toDouble(Object value) {
       return ((Number) value).doubleValue();
     }
 
+    @Override
+    public boolean toBoolean(Object value) {
+      return ((Number) value).intValue() > 0;
```

Comment:
Can object be null? Would null Object be considered true or false? What if we add NULL value support later?

Reply:
We won't get `null` here. (Usually we annotation arguments that can be `null` with `Nullable`)